data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorConfig.java

```diff
@@ -99,45 +121,71 @@ public class KeyValueProcessorConfig {
             "If the key was parsed from the source field that value will remain and the default value is not used. " +
             "If the default values includes keys which are not part of <code>include_keys</code> those keys and value will be added to the event.")
     @NotNull
+    @ExampleValues({
+        @Example(value = "{\"defaultkey\": \"defaultvalue\"}", description = "key1=value1 will parse into {\"key1\": \"value1\", \"defaultkey\": \"defaultvalue\"}.")
+    })
     private Map<String, Object> defaultValues = DEFAULT_DEFAULT_VALUES;
 
     @JsonProperty("non_match_value")
     @JsonPropertyDescription("Configures a value to use when the processor cannot split a key-value pair. " +
             "The value specified in this configuration is the value used in <code>destination</code> map. " +
             "The default behavior is to drop the key-value pair.")
+    @ExampleValues({
+        @Example(value = "none", description = "key1value1&key2=value2 parses into {\"key1value1\": \"none\", \"key2\": \"value2\"}.")
+    })
     private Object nonMatchValue = DEFAULT_NON_MATCH_VALUE;
 
     @JsonProperty(value = "include_keys", defaultValue = "[]")
     @JsonPropertyDescription("An array specifying the keys that should be included in the destination field. " +
             "By default, all keys will be added.")
     @NotNull
+    @ExampleValues({
+        @Example(value = "[\"key2\"]", description = "key1=value1&key2=value2 will parse into {\"key2\": \"value2\"}.")
+    })
     private List<String> includeKeys = DEFAULT_INCLUDE_KEYS;
 
     @JsonProperty(value = "exclude_keys", defaultValue = "[]")
     @JsonPropertyDescription("An array specifying the parsed keys that should be excluded from the destination field. " +
             "By default, no keys will be excluded.")
     @NotNull
+    @ExampleValues({
+        @Example(value = "[\"key2\"]", description = "key1=value1&key2=value2 will parse into {\"key1\": \"value1\"}.")
+    })
     private List<String> excludeKeys = DEFAULT_EXCLUDE_KEYS;
 
     @JsonPropertyDescription("A prefix to append before all keys. By default no prefix is added.")
+    @ExampleValues({
+        @Example(value = "custom", description = "{\"key1=value1\"} parses into {\"customkey1\": \"value1\"}.")
```

Comment:
Maybe it would be appropriate to have something like: `query:`?

Reply:
do you mean replacing `custom` with `query:` here?